spec/models/assignment_spec.rb

```diff
@@ -12,12 +12,6 @@
     it { is_expected.to have_many(:section_due_dates) }
     it { is_expected.to accept_nested_attributes_for(:section_due_dates) }
     it { is_expected.to have_one(:assignment_stat).dependent(:destroy) }
-    it do
-      is_expected.to have_many(:rubric_criteria).dependent(:destroy).order(:position)
```

Comment:
You should add in a new test for `is_expected.to have_many(:criteria)` with the same options, as well as `ta_criteria` and `peer_criteria` associations.